app/src/components/pages/Home.tsx

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
+import { useNavigate, NavigateFunction } from 'react-router-dom';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
 
 function Home() {
-  return <div>Home</div>;
+  let navigate: NavigateFunction = useNavigate();
+
+  return (
+    <Box
+      sx={{

```

Comment:
Are all these styling rules necessary?

Reply:
They centre the login button on the home page, makes it more appealing for the demo.  You can delete it and re-render to see what I mean.